src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx

```diff
@@ -127,6 +127,49 @@ You can request to increase the limits based on your plan. Contact the [technica
 
 <Button href="https://www.azion.com/en/pricing/#EdgeApplication" text="see pricing"></Button>
 
+### Total Data Transferred
+
+Total volume of data transferred by Azion Edge Computing Platform, whether it's delivered through cache or fetched from an origin, calculated in GB. Considers data passing through:
+
+- All customer's domains.
+- Users and Azion's Edge Servers.
+- Origin servers, even if the origin belongs to the Azion Network, such as by using Image Processor or Live Ingest services.
+- Edge Application and any of its modules and features enabled.
+
+There are several different scenarios in which data is transferred through Azion. For example:
+
+- Content delivered using an edge functions
+
+When an edge application delivers content created through an edge function on Azion, the request flow is processed fully through Azion servers and delivered directly to users.
+
+![Overview of a request/response flow using Edge Functions](/assets/docs/images/uploads/data-transferred-edge-functions)
+
+- Content offloaded through cache
+
+When content is found in Azion's cache, meaning there was a HIT in Edge Cache, it's delivered directly by the edge, without searching for the content on the origin. For clients using Tiered Cache, if there's a MISS in the Edge Cache Layer but a HIT in the Tiered Cache layer, the content is still delivered directly by the edge.
+
+Content hosted through Object Storage can also be stored in cache and be delivered directly by the edge.
+
+![Overview of a request/response flow of HIT in Edge Cache](/assets/docs/images/uploads/data-transferred-hit-cache)
+
+- Content feteched from an origin
+
+When content isn't found in Azion's cache, meaning there was a MISS in Edge Cache, Azion searches and fetches the content from the origin to deliver it to the user. For clients using Tiered Cache, there can be a MISS in both the Edge Cache and the Tiered Cache layer, and the content is searched on the origin before being delivered by the edge.
+
+For clients using Object Storage, Azion can also search and fetch the content from the origin if there's a MISS in Edge Cache. In this case, the origin is the object storage that was configured in the product.
+
+![Overview of a request/response flow of MISS in Edge Cache](/assets/docs/images/uploads/data-transferred-miss-cache-image-processor)
+
+- Content using Image Processor fetched from an origin
+
+Similar to the **Content offloaded through cache** and **Content feteched from an origin** flows, when the Image Processor module is enabled, images can be searched on Edge Cache and Tiered Cached, if enabled, and in case of a miss, the image is fetched from the origin and delivered by the edge, and cached for future requests.
```

Comment:
```suggestion
Similar to the **Content offloaded through cache** and **Content fetched from an origin** flows, when the **Image Processor** module is enabled, images can be searched on **Edge Cache** and **Tiered Cached**, if enabled, and in case of a miss, the image is fetched from the origin and delivered by the edge, and cached for future requests.
```